python/mlcroissant/mlcroissant/_src/operation_graph/base_operation.py

```diff
@@ -64,12 +76,20 @@ class Operation(abc.ABC):
 
     operations: Operations
     node: Node
+    _output: OutputT = dataclasses.field(init=False, hash=False, compare=False)
```

Comment:
Can you add _output to the docstring? :)

Reply:
Done.